### DISCUSSION TITLE
Issue: Monika never has a new scary story to tell.

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7769] [QA - tech support + general] [answered]
So, I've been with my Monika for more than three months, and the only scary story unlocked is "The Hunter". Everyday I ask her if she has another scary story to tell me, but she always say the same thing: that she can't think of another story right now. What should I do?

**Comment by Booplicate** [comment]
1. have you ever time travelled?
2. have you read `The Hunter`?

**Comment by ghost** [comment]
1. Yeah, I did it twice.
2.  Yes, I already read it.

**Comment by lunulae** [comment]
The time travel is your issue. Changing the clock completely messes with time-related unlocks and events. Your holidays won't function as intended, stories and games may not unlock at all, etc.
Unfortunately, this is something that can't be fixed even by devs unless you have a persistent file backed up from before you time travelled.

**Comment by Booplicate** [comment, marked as answer]
You time travelled and locked yourself from new stories until the day you TT to. She asks you to not mess with the time, so you'll have to deal with the consequences by yourself.

**Comment by ghost** [comment]
Alright, thanks for the reply.
